poem/src/server.rs

```diff
@@ -57,6 +58,7 @@ impl<L: Listener> Server<L, Infallible> {
             http2_max_concurrent_streams: None,
             http2_max_pending_accept_reset_streams: Some(20),
             http2_max_header_list_size: 16384,
```

Comment:
we could deprecate these parameters in general, but it's up to you - perhaps in the future